src/equations/compressible_euler_2d.jl

```diff
@@ -606,9 +606,9 @@ end
 # Calculate 1D flux for a single point
 @inline function flux(u, orientation::Integer, equations::CompressibleEulerEquations2D)
   rho, rho_v1, rho_v2, rho_e = u
-  v1 = rho_v1/rho
-  v2 = rho_v2/rho
-  p = (equations.gamma - 1) * (rho_e - 1/2 * rho * (v1^2 + v2^2))
+  v1 = rho_v1 / rho
+  v2 = rho_v2 / rho
+  p = (equations.gamma - 1) * (rho_e - 0.5 * (rho_v1 * v1 + rho_v2 * v2))
```

Comment:
Is preferring `0.5` over `1/2` a matter of style or performance? Just wondering...

Reply:
Just style - this way, it's more obvious that we're creating a `Float64` and didn't forget to add a second `/` to create a `Rational{Int}` or something like that.

Reply:
Thanks for the clarification!